migrate/Cargo.toml

```diff
@@ -1,13 +1,18 @@
 [package]
-name = "stress"
+name = "migrate"
```

Comment:
A bit of git screwup. This should be reviewed as a new file, not a rename. `bench/core` does not exist anymore.